Gir.toml

```diff
@@ -1868,6 +1868,15 @@ status = "generate"
     name = "get_path"
         [object.function.return]
         nullable = false
+    [[object.function]]
+    name = "add_events"
+    ignore = true
+    [[object.function]]
+    name = "get_events"
+    ignore = true
+    [[object.function]]
+    name = "set_events"
+    ignore = true
```

Comment:
property "events" need ignore too.

Reply:
Why? One might want to use the notify signal of it

Reply:
You right, I missed that `connect_property_events_notify` already generated.
And we don't have way to generate only notify without get/set 😭

Reply:
@EPashkin: I wonder why too. :)

Reply:
Ah damn. Github refreshed after I commented. Now we know why. 😆